packages/design-system/src/directives/n8n-truncate.ts

```diff
@@ -0,0 +1,11 @@
+import type { DirectiveBinding, ObjectDirective } from 'vue';
+import { truncate } from '../utils/string';
+
+export const n8nTruncate: ObjectDirective = {
```

Comment:
What does this do? A documentation + `@example` would be nice

Reply:
Added